Aptoide/bds-sdk
# [APPC-1740][APPC-1746] Pay as Guest - Check for ongoing transactions

**What does this PR do?**

Adds a check for ongoing transactions before loading the payment methodss
Fixes bug on rotation while loading payment methods

**Database changed?**

   No

**Where should the reviewer start?**

PaymentMethodsPresenter.java

**How should this be manually tested?**

Be very fast and kill the app before the transaction completes and try to make another purchase and check if the it retakes the previous transaction.
Use charles to mock transaction (you need to have done a transaction before, otherwise it will not return a purchase after) to return PROCESSING and check if it redirects and completes the purchase.


**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-1740
**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 




**Code Review Checklist**
